modules/efk-logging-elasticsearch-add-remove.adoc

```diff
@@ -32,46 +28,26 @@ nodes:
   - nodeCount: 2 <1>
     nodeSpec:
       resources: {}
-    roles:  <2>
```

Comment:
can we leave the roles as they are here since that is currently how CLO will create them?

Reply:
@ewolinetz  The CLO will create client/data/master nodes when scaling?
roles: 
   - client
    - data
    - master